src/server/server_family.cc

```diff
@@ -804,6 +804,18 @@ void AppendMetricWithoutLabels(string_view name, string_view help, const absl::A
   AppendMetricValue(name, value, {}, {}, dest);
 }
 
+// 'tuple'-like structure with named fields for command statistics.
+struct CommandStat {
+  CommandStat(uint64_t calls, uint64_t sum) : calls(calls), sum(sum) {
+    avg = sum / calls;
+  }
+  ~CommandStat() = default;
+
+  uint64_t calls;
+  double sum;
+  double avg;
```

Comment:
i would remove avg, no need to keep it. Also, what's the reason for using double for sum?

Reply:
avg can be a const method

Reply:
Originally I used double because I had some issues with resolution, thinking that it was being rounded from being an integer. I'll implement these changes.